starknet_py/net/gateway_client.py

```diff
@@ -205,10 +206,15 @@ async def get_transaction_receipt(self, tx_hash: Hash) -> TransactionReceipt:
 
     async def estimate_fee(
         self,
-        tx: AccountTransaction,
+        tx: Union[AccountTransaction, List[AccountTransaction]],
         block_hash: Optional[Union[Hash, Tag]] = None,
         block_number: Optional[Union[int, Tag]] = None,
-    ) -> EstimatedFee:
+    ) -> Union[EstimatedFee, List[EstimatedFee]]:
+        if isinstance(tx, List):
```

Comment:
```suggestion
        if isinstance(tx, list):
```
not sure about it, but just to be consistent with other places